x-pack/plugins/alerting/server/rules_client_factory.ts

```diff
@@ -133,6 +137,30 @@ export class RulesClientFactory {
         return eventLog.getClient(request);
       },
       eventLogger: this.eventLogger,
+      async isAuthenticationTypeAPIKey() {
+        if (!securityPluginStart) {
+          return false;
+        }
+        const user = await securityPluginStart.authc.getCurrentUser(request);
+        return user && user.authentication_type ? user.authentication_type === 'api_key' : false;
+      },
+      async getAuthenticationAPIKey(name: string) {
```

Comment:
I don't believe either of these methods needs to be async, technically.  I'm seeing a _squiggle_ under the `await` for `securityPluginStart.authc.getCurrentUser(request)` indicating _'await' has no effect on the type of this expression_.

Reply:
Oh my bad, I will fix this!

Reply:
Resolved in this commit https://github.com/elastic/kibana/pull/154189/commits/bc37396a966b90e9a37fc2b1039be91dd1403c89